### ISSUE TITLE
Erroneously fails to transcribe valid examples

### ISSUE THREAD

**Issue by eddieantonio** [eddieantonio/cree-sro-syllabics#18]
See the attached file. Not all examples are false positives (things that `sro2syllabics()` _should_ transcribe).

[untranslatedlemmas.txt](https://github.com/eddieantonio/cree-sro-syllabics/files/2877028/untranslatedlemmas.txt)

**Comment by eddieantonio** [comment]
Investigating further, a lot of these are definitely wrong:

| Erroneous form | Reason |
|-|-|
| acimostamaew | invalid vowel combination: 'ae' |
| ahtapw | invalid coda: 'pw' |
| mitsoh | should spell 'ts' as 'c' |
| ikatestaa | double vowel: probably meant 'â' instead of 'aa' |
| isisijmaw | invalid consonant: 'j' is probably supposed to be 'y' |
| khipa | invalid consonant cluster: 'kh' |
| kipostahikkew | invalid consonant cluster: 'kk' |
| kipistaneqw | literally what: 'qw' |
| kisakta | invalid cluster: 'kt'; perhaps should be 'kisakita' |

At least one I'm confused about:

| Form | Confusion |
| - | - |
| apihlow | rare letter: 'l' |

Others are definitely correct, and should be converted; for example, "mitêh" should be rendered as «ᒥᑌᐦ», but is not converted in the current version.

Paging Dr. @aarppe for help.

**Comment by eddieantonio** [comment]
Update: attached are valid forms (mostly forms that have an 'h' final) and invalid or unsure forms.

[invalid.txt](https://github.com/eddieantonio/cree-sro-syllabics/files/2877251/invalid.txt)
[valid.txt](https://github.com/eddieantonio/cree-sro-syllabics/files/2877252/valid.txt)

**Comment by aarppe** [comment]
Yup - throughout the MD recording project, we have noticed quite a deal of errors (those errors are apparent in the on-line Cree dictionary as well, AFAIR), and every time I've gone through the source for the aggregation, I've run into some typo, also in the English translation.

In particular, when scripting to standardize the orthography of the Maskwacîs head words, in the set of words that couldn't be analyzed with even the descriptive FST (1000-2000 entries), I ran into typos or non-standard spelling of the above types. There's actually a number of <ts> instead of <ci> in MD. Some of those should be pretty straight-forward to correct, but others need a manual consideration of the written form and the English translation, in order to be able to determine what would be the appropriate standardized form.

Hitherto, I've been wary of changing the MD source any more than absolutely necessary, and haven't wanted to do that piecemeal but rather in one concerted go - but on the intermediate term we certainly will have fix those (and I presume that is necessary for having a proper syllabic version of MD, which didn't occur to me earlier).
